code/modules/mob/living/simple_animal/hostile/mining_mobs/basilisk.dm

```diff
@@ -56,6 +46,24 @@
 		if(3)
 			adjustBruteLoss(110)
 
+/mob/living/simple_animal/hostile/asteroid/basilisk/OpenFire()
+	Gaze()
+	ranged_cooldown = world.time + ranged_cooldown_time
+
+/mob/living/simple_animal/hostile/asteroid/basilisk/proc/Gaze()
+	if(!isliving(target))
+		return
+	var/mob/living/L = target
+	if(L.has_status_effect(STATUS_EFFECT_PETRIFICATION) || L.incapacitated())
+		return
+	visible_message("<span class='warning'>[src] [ranged_message] [target]</span>")
+	to_chat(L, "<span class='boldannounce'>Your body starts to lock up as you look at [src]!</span>")
+	if(gaze_sound)
+		playsound(src, gaze_sound, 75, FALSE)
+	L.apply_status_effect(STATUS_EFFECT_PETRIFICATION)
```

Comment:
this should just be `L.apply_status_effect(STATUS_EFFECT_PETRIFICATION, src)` with the on_creation properly setting the petrifier it gets as an argument.